package.json

```diff
@@ -673,6 +673,17 @@
             "default": "true",
             "description": "%configuration.maven.dependency.enableConflictDiagnostics%",
             "scope": "window"
+          },
+          "maven.defaultOpenProjectMethod": {
```

Comment:
maven.projectOpenBehavior

Reply:
I directly followed spring-initializr, see https://github.com/microsoft/vscode-spring-initializr/pull/157

A better name in my mind was `maven.project.postCreationBehavior"

Reply:
I'm referring from https://github.com/microsoft/vscode-azurefunctions/blob/54c7614fbb984fbf3ff6bff1028979bd96f3a644/package.json#L952

postCreation sounds like a lifecycle method name but not a setting name

Reply:
then let's follow azfunc